ios/Podfile

```diff
@@ -1,21 +1,42 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, '11.0'
+platform :ios, '12.0'
 
 target 'BerbixSdk' do
   source 'https://cdn.cocoapods.org/'
   source 'https://github.com/berbix/berbix-ios-spec.git'
 
-  pod 'Berbix', '3.0.4'
+  pod 'Berbix', '3.1.0'
   
   use_frameworks!
 
   config = use_native_modules!
 
+  # Flags change depending on the env values.
+  #flags = get_default_flags()
+
   use_react_native!(
     :path => config[:reactNativePath],
     # to enable hermes on iOS, change `false` to `true` and then install pods
-    :hermes_enabled => false
+    :hermes_enabled => false,
+    :fabric_enabled => false, #flags[:fabric_enabled],
+    # Enables Flipper.
+    #
+    # Note that if you have use_frameworks! enabled, Flipper will not work and
+    # you should disable the next line.
+    #:flipper_configuration => FlipperConfiguration.enabled,
```

Comment:
should we put a note about this in the README so that folks know that they may need to modify this setting?